WordPress/src/main/java/org/wordpress/android/ui/reader/actions/ReaderPostActions.java

```diff
@@ -82,43 +83,52 @@ public static boolean performLikeActionLocal(final ReaderPost post,
         return true;
     }
 
-    public static void performLikeActionRemote(final ReaderPost post,
+    public static void performLikeActionRemote(@Nullable final ReaderPost post,
                                                final boolean isAskingToLike,
                                                final long wpComUserId,
-                                               final ActionListener actionListener) {
+                                               @Nullable final ActionListener actionListener) {
+        if (post != null) {
+            performLikeActionRemote(post, post.blogId, post.postId, isAskingToLike, wpComUserId, actionListener);
+        }
+    }
+
+    public static void performLikeActionRemote(
```

Comment:
Use an overload `performLikeActionRemote()` function to handle the nullalbe `ReaderPost`